src/views/ont/ONTRunIndex.vue

```diff
@@ -41,10 +41,8 @@
 import DataFetcher from '@/components/DataFetcher.vue'
 import DownloadIcon from '@/icons/DownloadIcon.vue'
 import FilterCard from '@/components/FilterCard.vue'
-import useQueryParams from '@/composables/useQueryParams'
-
-import { createNamespacedHelpers } from 'vuex'
-const { mapActions, mapGetters } = createNamespacedHelpers('traction/ont')
+import useQueryParams from '@/composables/useQueryParams.js'
+import useOntRootStore from '@/stores/ontRoot.js'
 
 export default {
```

Comment:
This file looks different compared to `src/views/pacbio/PacbioPoolIndex.vue`. I believe both are using Pinia. Is it because `src/views/pacbio/PacbioPoolIndex.vue` is using new Vue features or is that the other way around?

Reply:
Both components are now using the Pinia store, but previously, OnRunIndex was using the VueX store for the ontRoot store. The difference between the two components is that PacbioPoolIndex uses a different syntax for Vue 3. PacbioPoolIndex was recently migrated from Vue 2 to Vue 3, so it uses the more modern <script setup>, which is a compiler macro that allows for less boilerplate code and includes improvements like automatic imports, among others.

Reply:
Interesting. Looks like it injects a bunch of code. Anyway thanks!